services/src/layers/layer.rs

```diff
@@ -29,13 +31,15 @@ pub struct Layer {
     pub description: String,
     pub workflow: Workflow,
     pub symbology: Option<Symbology>,
+    pub properties: Option<HashMap<String, String>>,
 }
 
 #[derive(Debug, Serialize, Deserialize, Clone, PartialEq, Eq)]
 pub struct LayerListing {
     pub id: ProviderLayerId,
     pub name: String,
     pub description: String,
+    pub properties: Option<HashMap<String, String>>,
```

Comment:
it's because in the collection item listing the properties are stuff that is rendered as strings as part of the listing. In the LayerListing it contains stuff that it isn't displayed but used for actually loading the data, like timeSteps. There, I need to access the stuff based on the key instead of just rendering it (in order). Maybe we should call it differently here. metadata?